examples/radix-example-vite-spa-env-variables/Dockerfile

```
@@ -0,0 +1,25 @@
+FROM node:18-alpine3.16 AS build
+
+WORKDIR /app
+
+ENV PATH=/app/node_modules/.bin:$PATH
+
+# install app dependencies
+COPY ./package.json ./
+COPY ./package-lock.json ./
+
+RUN npm ci
+
+COPY . ./
+
+RUN npm run build
+
+FROM nginxinc/nginx-unprivileged:stable-alpine
+COPY --chown=101 --from=build /app/dist /usr/share/nginx/html
+
+COPY ./100-inject-envvars.sh /docker-entrypoint.d/100-inject-envvars.sh
```

This was required to make it work on my machine :) 

```suggestion
COPY --chmod=755 ./100-inject-envvars.sh /docker-entrypoint.d/100-inject-envvars.sh
```